### ISSUE TITLE
Budget section for competitions

### ISSUE THREAD

**Issue by viroulep** [thewca/worldcubeassociation.org#5465]
See [this project](https://github.com/thewca/worldcubeassociation.org/projects/12) for the general idea.
The first step of that project would be to introduce a "budget" section for each competition (that could be introduced in the "edit" submenu").
This section should be accessible to only:
  - admin
  - WEC
  - WCAT
  - Board?

It should allow two separate uploads:
  - one "provisional" budget
  - one "final" budget

The provisional one should be editable as long as the competition is not confirmed.

The final one should be submitted only after competition is done (we could re-use the same check as we use for posting competition reports), and should not be editable once submitted.
The members from committees above should be able to remove an existing final budget to allow re-submission in case the Delegate uploaded an incorrect file.

**Comment by c-goodyear** [comment]
Relevant people needing access also includes WFC.

**Comment by EdHollingdale** [comment]
I assume the applicable Senior Delegate would also have access to view the draft and final budgets?

**Comment by c-goodyear** [comment]
Yes, I thought "admin" contained the listed Delegates, organizers, and relevant Senior.

**Comment by viroulep** [comment]
Actually admin is just WST, I missed the obvious and "default" authorized people: the competition managers!
I fixed the OP.

**Comment by Jambrose777** [comment]
> Actually admin is just WST, I missed the obvious and "default" authorized people: the competition managers!
> I fixed the OP.

competition managers includes:
WST, Board, WRT, WCAT, organizers of comp, Delegates of comp, WRC, Seniors of comp, WEC

**Comment by moralsh** [comment]
I'm going to give this a try

**Comment by saranshgrover** [comment]
@moralsh Are you still working on this?
